test/helpers/MakeGnosisWallet.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import "forge-std/console.sol";
+import "forge-std/Test.sol";
+
+import { GnosisSafeL2 } from "safe-global/safe-contracts/GnosisSafeL2.sol";
+import { GnosisSafeProxyFactory } from "safe-global/safe-contracts/proxies/GnosisSafeProxyFactory.sol";
+import { Enum } from "safe-global/safe-contracts/common/Enum.sol";
+import { DefaultCallbackHandler } from "safe-global/safe-contracts/handler/DefaultCallbackHandler.sol";
+//import { CompatibilityFallbackHandler } from "safe-global/safe-contracts/handler/CompatibilityFallbackHandler.sol";
+
+library MakeGnosisWallet {
```

Comment:
should we remove this lib?

Reply:
why would you? it's super helpful to setup a gnosis wallet for tests...